### PR TITLE
Add support for boxwidth.

### PR DESCRIPTION
**Pull request by lbarasti**
See gnuplot [documentation](http://gnuplot.sourceforge.net/docs_4.2/node163.html) on boxwidth.

### CODE REVIEW

**Comment by lbarasti** [comment]
Hi, I hope you don't mind me opening this.

Things that I'd be happy to discuss:
* I've omitted clearing `boxwidth` in `Chart#clear`, as similar properties - e.g. `view` - are also not cleared.
* I've added a box plot example with related png, but I have not added the example to the README.
* boxwidth is `absolute` by default. My changes don't cover setting boxwidth to `relative`. I thought this can be added later, as needed. I also wasn't sure about what the parameters would look like.

File: src/ishi/base.cr [hunk 1]
```diff
@@ -223,6 +223,13 @@ module Ishi
       self
     end
 
+    # set the default width of boxes in the boxes, boxerrorbars, candlesticks
```
Comment (toddsundsted) [inline src/ishi/base.cr]: Could you capitalize this comment and add a link to the documentation to match the other sections. (This is the correct link, I believe: http://www.gnuplot.info/docs_5.2/Gnuplot_5.2.pdf#section*.232)

**Review by toddsundsted — COMMENTED** [review]
This is great! I appreciate the PR. I have one small requested change.

File: src/ishi/base.cr (hunk 1 above)
Comment (lbarasti) [reply src/ishi/base.cr]: Done!

**Review by toddsundsted — APPROVED** [review]

**Comment by toddsundsted** [comment]
@lbarasti thanks!

**Pull request merged**